From 28a68b65afb5340337ef89e635106d12a4d9b56b Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:26 -0400
Subject: [usb] ohci: always register debug files

Message-id: <1410887463-20674-195-git-send-email-dzickus@redhat.com>
Patchwork-id: 95157
O-Subject: [RHEL7 PATCH 194/591] ohci:always register debug files
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 1714ba0e8ec07ff9a7f6d165d4fbb471df6c10da
Author: Oliver Neukum <oneukum@suse.de>
Date:   Mon Nov 18 13:22:59 2013 +0100

    ohci:always register debug files

    Just remove the conditional compilation.

    Signed-off-by: Oliver Neukum <oneukum@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/ohci-hcd.c b/drivers/usb/host/ohci-hcd.c
index 18cc500..40cf076 100644
--- a/drivers/usb/host/ohci-hcd.c
+++ b/drivers/usb/host/ohci-hcd.c
@@ -1270,13 +1270,11 @@ static int __init ohci_hcd_mod_init(void)
   sizeof (struct ed), sizeof (struct td));
  set_bit(USB_OHCI_LOADED, &usb_hcds_loaded);
 
-#ifdef DEBUG
  ohci_debug_root = debugfs_create_dir("ohci", usb_debug_root);
  if (!ohci_debug_root) {
   retval = -ENOENT;
   goto error_debug;
  }
-#endif
 
 #ifdef PS3_SYSTEM_BUS_DRIVER
  retval = ps3_ohci_driver_register(&PS3_SYSTEM_BUS_DRIVER);
@@ -1431,11 +1429,9 @@ static int __init ohci_hcd_mod_init(void)
  ps3_ohci_driver_unregister(&PS3_SYSTEM_BUS_DRIVER);
  error_ps3:
 #endif
-#ifdef DEBUG
  debugfs_remove(ohci_debug_root);
  ohci_debug_root = NULL;
  error_debug:
-#endif
 
  clear_bit(USB_OHCI_LOADED, &usb_hcds_loaded);
  return retval;
@@ -1489,9 +1485,7 @@ static void __exit ohci_hcd_mod_exit(void)
 #ifdef PS3_SYSTEM_BUS_DRIVER
  ps3_ohci_driver_unregister(&PS3_SYSTEM_BUS_DRIVER);
 #endif
-#ifdef DEBUG
  debugfs_remove(ohci_debug_root);
-#endif
  clear_bit(USB_OHCI_LOADED, &usb_hcds_loaded);
 }
 module_exit(ohci_hcd_mod_exit);
-- 
1.7.1